test/unit/presenters/publishing_api/case_study_presenter_test.rb

```diff
@@ -64,7 +64,7 @@ def present(edition)
       presented_item.content[:details].delete(:body)
     )
     assert_equal expected_content[:details], presented_item.content[:details].except(:body)
-    assert_equal expected_links, presented_item.links
+    assert_hash_includes presented_item.links, expected_links
```

Comment:
Yeah, fair question. I suppose the thinking is that a test should only test what it cares about - in this case that the `expected_links` are present. While there _could_ be instances that you want to make sure that the payload matches exactly, I can't really think of a thing that could wrong here. Especially because all the payloads are tested against the content schemas.